src/celeritas/em/xs/MottRatioCalculator.hh

```diff
@@ -9,8 +9,8 @@
 #include "corecel/Macros.hh"
 #include "corecel/Types.hh"
 #include "corecel/math/ArrayUtils.hh"
+#include "corecel/math/PolyEvaluator.hh"
 #include "celeritas/em/data/WentzelOKVIData.hh"
-#include "celeritas/grid/PolyEvaluator.hh"
```

Comment:
Given the number of files this changes, would it be easy enough for you to turn this rename into a trivial PR?

Reply:
No problem! If you want I could also split the tridiagonal solver into separate small PR.

Reply:
Only if you want more lines devoted to yourself in the release notes 😉

Reply:
Yes gimme more lines 🤑

Reply:
More lines, more 💰